client/src/containers/pages/locations/LocationUnitGroup/index.tsx

```diff
@@ -0,0 +1,286 @@
+    const currentValue = e.target.value;
+    setValue(currentValue);
+    const filteredData = tableData.filter((entry) =>
+      entry.name.toLowerCase().includes(currentValue.toLowerCase())
+    );
+    setData(filteredData);
+  };
+
+  return (
+    <section>
+      <Helmet>
+        <title>Locations Unit</title>
+      </Helmet>
+      <Row justify="start" className="weclome-box">
+        <Col span={24}>
+          <h5>Location Unit Group Management</h5>
+        </Col>
+      </Row>
+      <Row>
+        <Col span={selectedLocation !== null ? 16 : 24}>
+          <Row className="bg-white">
+            <Col span={24}>
+              <div className="mb-3 mt-3 mr-1 ml-3 d-flex justify-content-between">
+                <h5>
+                  <Input
+                    placeholder="Search"
+                    size="large"
+                    value={value}
+                    prefix={<SearchOutlined />}
+                    onChange={onChange}
+                  />
+                </h5>
+                <div>
+                  <Button type="primary">
+                    <PlusOutlined />
+                    Add location unit group
+                  </Button>
+                  <Divider type="vertical" />
+                  <Dropdown
+                    overlay={
+                      <Menu>
+                        <Menu.Item key={'1'}>Logout</Menu.Item>
+                      </Menu>
+                    }
+                    placement="bottomRight"
+                  >
+                    <Button shape="circle" icon={<SettingOutlined />} type="text" />
+                  </Dropdown>
+                </div>
+              </div>
+              <div className="table-container">
+                <Form form={form} component={false}>
+                  <Table
+                    components={{ body: { cell: EditableCell } }}
+                    dataSource={data}
+                    columns={mergedColumns}
+                    rowClassName="editable-row"
+                    pagination={{ onChange: cancel, showQuickJumper: true }}
+                  />
+                </Form>
+              </div>
+            </Col>
+          </Row>
+        </Col>
+        {selectedLocation !== null ? (
+          <Col className="pl-3 border-left" span={8}>
+            <LocationDetail onClose={() => setSelectedLocation(null)} {...selectedLocation} />
+          </Col>
+        ) : null}
+      </Row>
+    </section>
+  );
+};
+
+export { LocationUnitGroup };
+
+/** Connect the component to the store */
+
+/** map state to props */
+
+// const mapStateToProps = (state: Partial<Store>) => {
+//   const result = {
+//     extraData: getExtraData(state),
+//   };
+//   return result;
+// };
+
+/** create connected component */
+
+/** Connected Header component
+ */
+const ConnectedLocationUnitGroupComponent = connect(null, null)(LocationUnitGroup);
+
+export default ConnectedLocationUnitGroupComponent;
```

Comment:
@HaseemUlHaq should we move connected component to different file ? as i have observed that with other components